src/components/inputs/date-time-input/date-time-input.story.js

```diff
@@ -13,41 +12,38 @@ storiesOf('Inputs', module)
   .addDecorator(withKnobs)
   .addDecorator(withReadme(Readme))
   .add('DateTimeInput', () => {
-    const locale = select('locale', ['de', 'en'], 'en');
```

Comment:
This is one of the work-around cases where `locale` was passed as a key to force the component to rerender. The new input components will no longer need this workaround, so I removed it.

This means they date inputs will ignore the changing locale in stories (as they would in features) until the new date inputs replace them.